django_loci/static/django-loci/js/loci.js

```diff
@@ -327,8 +330,9 @@ django.jQuery(function ($) {
     $('#content-main form').submit(function (e) {
         var indoorPosition = $('.field-indoor .floorplan-raw input').val();
         if ($type.val() === 'indoor' && !indoorPosition) {
-            e.preventDefault();
-            alert(gettext('Please set the indoor position before saving'));
+            var confirmation = confirm('Do you want to save this indoor' +
+                                    'location without indoor coordinates?');
+            if (!confirmation) { e.preventDefault(); }
```

Comment:
we should allow to save only if no floorplan is available.

If the floorplan is availabe, we must still force the user to specify the coordinates.